addon/components/frost-table-cell.js

```diff
@@ -18,7 +18,9 @@ export default Component.extend({
   // == PropTypes =============================================================
 
   propTypes: {
+    onCallback: PropTypes.func.required,
     cellRenderer: PropTypes.any,
+    colIndex: PropTypes.number,
```

Comment:
Similarly here, why does the cell need to know the column index? Well, I suppose right now it needs it to give it to the cell renderer.